SConstruct

```diff
@@ -137,6 +137,7 @@ opts.Add(BoolVariable("vsproj", "Generate a Visual Studio solution", False))
 opts.Add(EnumVariable("macports_clang", "Build using Clang from MacPorts", "no", ("no", "5.0", "devel")))
 opts.Add(BoolVariable("disable_3d", "Disable 3D nodes for a smaller executable", False))
 opts.Add(BoolVariable("disable_advanced_gui", "Disable advanced GUI nodes and behaviors", False))
+opts.Add(BoolVariable("modules_enabled_by_default", "If no, only build modules declared (e.g., in custom.py)", True))
```

Comment:
`custom.py` doesn't get special tratement from this specific option, so I recommend just stating this:

```suggestion
opts.Add(BoolVariable("modules_enabled_by_default", "If 'no', disable all modules except the ones explicitly enabled", True))
```